app/components/documents/document-card.hbs

```diff
@@ -23,34 +23,34 @@
                 {{/unless}}
               </span>
             </div>
-            <LinkTo @route="document" @model={{this.piece.id}}
-              class="auk-h4" data-test-document-card-name-value>
-              {{this.piece.name}}{{#if this.piece.file}}.{{this.piece.file.extension}}{{/if}}
+            <LinkTo
+              @route="document"
+              @model={{this.piece.id}}
+              class="auk-h4"
+              data-test-document-card-name-value
+            >
+              {{this.piece.name}}
             </LinkTo>
             <div class="vlc-document-card-content__meta">
               {{! no piece.created in legacy data }}
               {{#if this.piece.created}}
-                <p>
+                <p data-test-document-card-primary-source-created={{if this.piece.file.derived ""}}>
                   {{t "uploaded-at"}}
                   {{moment-format this.piece.created}}
                   {{t "at"}}
                   {{moment-format this.piece.created "HH:mm"}}
-                </p>
-              {{/if}}
-              {{#if this.piece.file.primarySource.created}}
-                <p data-test-document-card-primary-source-created>
-                  {{t "uploaded-at"}}
-                  {{moment-format this.piece.file.primarySource.created}}
-                  {{t "at"}}
-                  {{moment-format this.piece.file.primarySource.created "HH:mm"}}
-                  {{t "as"}}
-                  <AuLinkExternal
-                    data-test-document-card-primary-source-link
-                    href={{this.piece.file.primarySource.namedDownloadLink}}
-                    download
-                  >
-                    {{t (if (eq this.piece.file.primarySource.extension "docx") "word-document" "source-file-lowercase")}}
-                  </AuLinkExternal>
+                  {{#if this.piece.file.derived}}
+                    {{t "as"}}
+                    {{!-- TODO: AuLinkExternal has a font-size of 16px (because it uses the same size as buttons), but that's larger than normal text (which uses 14px).
```

Comment:
@brenner-company

Dit ziet er zo uit
![image](https://user-images.githubusercontent.com/22411874/200867779-0e079dc1-33cf-4881-aad8-2be11376a46b.png)

`bronbestand` is nogal groter dan de rest van de tekst. Dit komt volgens mij door [deze CSS rules](https://github.com/kanselarij-vlaanderen/frontend-kaleidos/blob/development/app/styles/au/_au-components.scss#L202-L206). Misschien is het interessant om de font-size enkel aan te passen voor `au-c-link.button-` klasses. Dus een gewone `AuLinkExternal` heeft wel dezelfde grootte als normale tekst, maar `AuLinkExternal` met een button skin is dan weer groter.